src/T2DMap.cpp

```diff
@@ -2082,9 +2082,17 @@ void T2DMap::paintEvent(QPaintEvent* e)
             TArea* _paid = mpMap->mpRoomDB->getArea(_iaid);
             QString _paid_name = mpMap->mpRoomDB->getAreaNamesMap().value(_iaid);
             if (_paid) {
-                infoText = tr("Area: %1 ID:%2 x:%3 <-> %4 y:%5 <-> %6 z:%7 <-> %8\n",
-                              "%1 is the (text) name of the area, %2 is the number for it, %3 to %8 are pairs (min <-> max) of extremes for each of x,y and z coordinates")
-                                   .arg(_paid_name,
+                infoText = tr("Area:%1%2 ID:%1%3 x:%1%4%1<‑>%1%5 y:%1%6%1<‑>%1%7 z:%1%8%1<‑>%1%9\n",
+                              // Intentional separator
+                              "This text uses non-breaking spaces (as '%1's, as Qt Creator cannot handle"
```

Comment:
What is this problem? `lupdate` has been fixed to handle C++11 string literals (finally, after 8 years...)

Reply:
Now reported by me as: https://bugreports.qt.io/browse/QTCREATORBUG-22052 ! :grinning: